data_steward/utils/slack_alerts.py

```diff
@@ -52,6 +71,23 @@ def _get_slack_channel_name():
     return os.environ[SLACK_CHANNEL]
 
 
+def _is_channel_available():
+    """
+    Test if the Slack channel is available
+    :return:
+    """
+    client = _get_slack_client()
+    channel_name = _get_slack_channel_name()
+    response = client.conversations_list(limit=2000)
+    if response.status_code == 200:
+        for channel in response.data['channels']:
+            if 'test' in channel['name']:
+                print(channel['name'])
+            if channel['name'] == channel_name:
+                return True
+    return False
```

Comment:
We might need to use try... except to catch `SlackApiError` if `client.conversations_list` fails to run. We don't want to raise `SlackApiError` in this case because the unit tests that import `validation.main.py` will fail because they don't have valid slack_token and slack_channel set up. 

```suggestion
def _is_channel_available():
    """
    Test if the Slack channel is available
    :return:
    """
    client = _get_slack_client()
    channel_name = _get_slack_channel_name()
    try:
        response = client.conversations_list(limit=sys.maxsize) -- use the max size allowed to avoid a hardcoded int
        if response.status_code == 200:
            for channel in response.data['channels']:
                if channel['name'] == channel_name:
                    return True
    except SlackApiError as e:
        logging.error(e)
    return False
```

Reply:
Gotcha! Before if there wasn't 200 response code returned from `conversations_list` it would return false raising all sorts of errors in unit test modules that depended on having the `SLACK_TOKEN` and `CHANNEL_NAME`.